colcon_bundle/verb/utilities.py

```diff
@@ -136,8 +145,14 @@ def update_symlinks(base_path):
                             # Create directory (permissions?)
                             os.makedirs(os.path.dirname(bundle_library_path),
                                         exist_ok=True)
-                        shutil.copy(symlink_dest_path,
-                                    bundle_library_path)
+                        if os.path.exists(symlink_dest_path):
```

Comment:
For the general use case of someone building and bundling in a development environment this should never be false, since the local system will have everything installed that is required to bundle the robot.

This will keep an exception from being thrown, but should we continue, or just exit with a better error message?

Reply:
I kinda like that it dies here right now, I'm conflicted on what the behavior should be.